plotly_resampler/figure_resampler/utils.py

```diff
@@ -4,7 +4,10 @@
 import pandas as pd
 
 from plotly.basedatatypes import BaseFigure
-from plotly.basewidget import BaseFigureWidget
+try:
+    from plotly.basewidget import BaseFigureWidget
```

Comment:
Why does this have to be in try-except?

Reply:
As a result we do not need Ipywidgets as a required dependency (which is useful when you want to just use it for a dash-app)